src/Components/TabsPage.tsx

```diff
@@ -0,0 +1,56 @@
+import classNames from 'classnames';
+import { Link } from 'react-router-dom';
+import { Tab } from '../types/Tab';
+
+type Props = {
+  tabs: Tab[],
+  selectedTabId: string | undefined,
+  setSelectedTab: (tab: Tab) => void
+};
+
+export const TabsPage: React.FC<Props> = ({
+  tabs, selectedTabId, setSelectedTab,
+}) => {
+  const isSelectedTab = tabs.find(tab => tab.id === selectedTabId) || undefined;
```

Comment:
```suggestion
  const isSelectedTab = !!tabs.find(tab => tab.id === selectedTabId);
```
`find` returns `undefined` by default no need to write it explicitly
better to transform it to boolean value